docs/toc.yml

```diff
@@ -524,7 +524,7 @@
       href: publish/package-your-add-in-using-visual-studio.md
     - name: Publish add-ins via Centralized Deployment
       href: publish/centralized-deployment.md
-    - name: Publish add-ins to a SharePoint catalog
+    - name: Publish add-ins to a SharePoint app catalog
```

Comment:
Suggest that you make this terminology update globally throughout the doc set. 

Based on a brief global search of the docs just now, it looks like there are a variety of ways this may be worded, so just doing a global search & replace for a term like "SharePoint catalog" likely won't be feasible. I'd probably suggest that you do a global search for "SharePoint" and then examine each result and update as appropriate...seems like that'd be the safest way to catch all occurrences.

Reply:
Okay, did a search and replace where it made sense. When in Office, like trust settings, the UI is add-in, so had to be careful on the replacing.